test/trento/application/projectors/host_projector_test.exs

```diff
@@ -166,6 +167,28 @@ defmodule Trento.HostProjectorTest do
                      1000
   end
 
+  test "should update the selected_checks field when event is received" do
+    %{id: host_id} = insert(:host, id: UUID.uuid4())
```

Comment:
Remove the `id: UUID.uuid4`. The factory is actually doing that

Reply:
Ah thanks!